src/incubator/Dialog/index.tsx

```diff
@@ -163,7 +164,7 @@ const Dialog = (props: DialogProps, ref: ForwardedRef<DialogImperativeMethods>)
   const renderDialog = () => {
     return (
       <PanGestureHandler onGestureEvent={isEmpty(directions) ? undefined : panGestureEvent}>
-        <View pointerEvents={'box-none'} reanimated style={style} onLayout={onLayout} ref={setRef} testID={testID}>
+        <View {...containerProps} reanimated style={style} onLayout={onLayout} ref={setRef} testID={testID}>
```

Comment:
Isn't removing `pointerEvents={'box-none'}` the fix? So not sure why the 'containerProps' are needed... Assuming that's for passing 'pointerEvents', in what scenario would the user need to pass `pointerEvents={'box-none'}` in the container props?

Reply:
In private we have a case were we render our own `header` which only part of it has `pointerEvents: 'box-none'` - that's the use case.

From the description:
> Do not set pointerEvents by default (allow via props) - this fixes a bug in Android where clicking on the header closes the dialog (it goes through to the fade view)

You can look at master in private on `ActionSheet` (Android only) - click on the header where there's no text